django_app_server_db_server/deployment/roles/app_server/tasks/main.yml

```diff
@@ -4,6 +4,7 @@
   yum: name={{ item }} state=present
   with_items:
     - git
+    - mercurial
```

Comment:
You probably want to attach a conditional here as well. We don't need `mercurial` on every install.